roles/confluent.ksql/tasks/rbac.yml

```
@@ -198,7 +193,7 @@
         "resourcePatterns": [
           {
             "resourceType":"Topic",
-            "name":"{{ksql_final_properties['ksql.service.id']}}processing_log",
+            "name":"{{ksql_final_properties['ksql.service.id']}}{{ksql_processing_log}}",
```

```suggestion
            "name":"{{ksql_final_properties['ksql.logging.processing.topic.name']}}",
```
Just dove more into this, we build out that property already. It might make more sense to just use the property